moped-editor/src/components/GridTable/Search.js

```diff
@@ -267,9 +277,10 @@ const Search = ({
               className={classes.gridSearchPadding}
             >
               <SearchBar
-                query={query}
-                searchState={searchState}
```

Comment:
Same here as with the filter state, `searchState` was an object containing both the state and state setter for the simple search term. I broke it out into two props.